qfieldsync/qfield_sync.py

```diff
@@ -145,6 +145,12 @@ def __init__(self, iface):
             self.cloud_item_gui_provider
         )
 
+        # autologin
+        if self.preferences.value("qfieldCloudRememberMe"):
+            dialog = CloudLoginDialog(self.network_manager)
+            dialog.authenticate()
```

Comment:
Would be nice to refactor this logic out of the dialog.

Reply:
Agree